templates/init-registry-org.py.j2

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+import json
+import logging
+import requests
+
+HELP_DEFINITION='''
+Script to initiate Gitea Image registry organization
+'''
+HELP_EXAMPLE='''
+Example:
+\n\t./init-regisitry-org.py 
+  --org org1,org2,org3 \
+  --url https://example.com/gitea \
+  --user username --password password
+'''
+
+headers= {
+    'Content-type': 'application/json'
+}
+
+data={
+    'visibility': 'limited'
+}
```

Comment:
That looks like a `visibility='limited'` argument for the `create_org` function, not a global var.